<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplSettingUI</class>
 <widget class="QWidget" name="TemplSettingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemplSettingUI</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1141</width>
     <height>861</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_segThresh_title">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>380</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>图像分割阈值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>560</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_modelType_title">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>590</y>
     <width>131</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>产品型号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_modelType">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>650</y>
     <width>191</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>780</y>
     <width>100</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>740</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>1</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>21</pointsize>
     <weight>12</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 100 21pt &quot;黑体&quot;;
</string>
   </property>
   <property name="text">
    <string>PCB检测系统 - 模板设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 242, 252);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modelType">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>650</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maskRoi_tl_x">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>160</y>
     <width>65</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_title">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>100</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>模板区域坐标</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_br_y">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>200</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_maskRoi_br">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>210</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_tl_x">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>160</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_br">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>200</y>
     <width>71</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>右下角</string>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>350</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_tl">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>160</y>
     <width>71</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>左上角</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_maskRoi">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>260</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_br_x">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>200</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_maskRoi_tl">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maskRoi_br_y">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>200</y>
     <width>65</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maskRoi_tl_y">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>160</y>
     <width>65</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_tl_y">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>160</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maskRoi_br_x">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>200</y>
     <width>65</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_segThresh">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>430</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>420</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>420</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_segThresh">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>480</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskRoi_br_2">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>470</y>
     <width>51</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阈值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_segThresh">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>470</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_segThresh">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>470</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>780</y>
     <width>100</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>graphicsView</zorder>
  <zorder>label_segThresh_title</zorder>
  <zorder>line_2</zorder>
  <zorder>label_modelType_title</zorder>
  <zorder>lineEdit_modelType</zorder>
  <zorder>pushButton_return</zorder>
  <zorder>line_3</zorder>
  <zorder>line_title</zorder>
  <zorder>label_title</zorder>
  <zorder>pushButton_modelType</zorder>
  <zorder>lineEdit_maskRoi_tl_x</zorder>
  <zorder>label_maskRoi_title</zorder>
  <zorder>label_maskRoi_br_y</zorder>
  <zorder>checkBox_maskRoi_br</zorder>
  <zorder>label_maskRoi_tl_x</zorder>
  <zorder>label_maskRoi_br</zorder>
  <zorder>line_1</zorder>
  <zorder>label_maskRoi_tl</zorder>
  <zorder>pushButton_maskRoi</zorder>
  <zorder>label_maskRoi_br_x</zorder>
  <zorder>checkBox_maskRoi_tl</zorder>
  <zorder>lineEdit_maskRoi_br_y</zorder>
  <zorder>lineEdit_maskRoi_tl_y</zorder>
  <zorder>label_maskRoi_tl_y</zorder>
  <zorder>lineEdit_maskRoi_br_x</zorder>
  <zorder>horizontalSlider_segThresh</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>checkBox_segThresh</zorder>
  <zorder>label_maskRoi_br_2</zorder>
  <zorder>lineEdit_segThresh</zorder>
  <zorder>pushButton_segThresh</zorder>
  <zorder>pushButton_confirm</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
